test/unit/collector/serverless.test.js

```diff
@@ -18,9 +18,9 @@ const fs = require('fs')
 const helper = require('../../lib/agent_helper')
 const API = require('../../../lib/collector/serverless')
 const serverfulAPI = require('../../../lib/collector/api')
-
-const pipePath = '/tmp/newrelic-telemetry'
-const shouldUsePipe = fs.existsSync(pipePath)
+const path = require('path')
+const NEWRELIC_PIPE_PATH = path.resolve('/tmp', 'newrelic-telemetry')
+const shouldUsePipe = fs.existsSync(NEWRELIC_PIPE_PATH)
```

Comment:
Seems like more of the dynamic situation where the test will run differently depending on the environment it is running in. Would prefer for this to be deterministic. Either setup to always use a pipe/certain path or always setup to use the old school way, or one of each.

Reply:
Agreed--I can remove these from the test file, since we're doing a one-off custom path for testing the same functionality.